packages/graphql/src/lib.ts

```diff
@@ -99,4 +101,4 @@ export const libDef = {
 
 export const $lib = createTypeSpecLibrary(libDef);
 
-export const { reportDiagnostic, createDiagnostic } = $lib;
+export const { reportDiagnostic, createDiagnostic, createStateSymbol } = $lib;
```

Comment:
I think Timothee suggested that we don't need `createStateSymbol`: https://github.com/microsoft/typespec/pull/5033#discussion_r1834801763

Reply:
Nice, this is a lot cleaner.